src/App.js

```diff
@@ -1,24 +1,28 @@
-import logo from './logo.svg';
-import './App.css';
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import { NOTES } from "config/constants";
+import { useEffect } from "react";
+
+import MyNotes from "./pages/MyNotes/MyNotes";
+import "./styles.css";
+
+const theme = createTheme({
+  palette: {
+    mode: "dark",
+  },
+});
 
 function App() {
+  useEffect(() => {
+    const json = JSON.stringify(NOTES);
+    localStorage.setItem("notes", json);
+  }, []);
+
```

Comment:
It's not a good practice to describe any kind of logic with internal components in `App` function. Maybe we can move it directly to the `NotesContainer`?